orbit/pkg/table/login_password/login_password.go

```diff
@@ -0,0 +1,67 @@
+//go:build darwin
+// +build darwin
+
+package login_password
+
+import (
+	"context"
+	"fmt"
+	"github.com/osquery/osquery-go/plugin/table"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.IntegerColumn("password_hint_enabled"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	uid, gid, err := getConsoleUidGid()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get console user: %w", err)
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+	cmd := exec.CommandContext(ctx, "dscl", ".", "-list", "/Users", "hint")
+
+	// Run as the current console user (otherwise we get empty results for the root user)
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		Credential: &syscall.Credential{Uid: uid, Gid: gid},
+	}
+
+	out, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("generate failed: %w", err)
+	}
+
+	res := "0"
+	if len(strings.TrimSpace(string(out))) > 0 {
+		res = "1"
+	}
+	return []map[string]string{{"password_hint_enabled": res}}, nil
+}
+
+// getActiveUserGroup gets the uid and gid of the current (or more accurately, most recently logged
```

Comment:
@zwass 

The such "most recently logged in" user may not have the permissions necessary for executing any given command, right?

That said, in this particular PR it looks like all users can execute `dscl . -list /Users hint` and see all other users hints (see below).

--

On a macOS Ventura I performed the following test:
1. Created two non-administrative (aka "Standard") users `foo` and `zoo`. 
User `foo` has `foobar` as pasword hint, and `zoo` has `zoobar` as password hint.
3. I logged in as `foo` and opened the terminal.
4. Ran the following command:
```
foo@luks-Mac ~ % dscl . -list /Users hint            
foo   foobar
zoo   zoobar
```

It's was weird to see that the user `foo` can see `zoo`'s password hint... but then I realized that the hint is sort of available for everyone, macOS shows it to you after the user fails to login three times (unless you disable such hint feature via another CIS check that we already implemented, #1873).

So... we are all good here :)